### PR TITLE
Banners

### PR DESCRIPTION
**Pull request by hamen**
Refers to https://github.com/JetBrains/jewel/issues/611

### CODE REVIEW

File: samples/standalone/src/main/kotlin/org/jetbrains/jewel/samples/standalone/view/component/Banners.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.jewel.samples.standalone.view.component
+
+import androidx.compose.runtime.Composable
+
+@Composable
+internal fun Banners() {
+}
```
Comment (rock3r) [inline samples/standalone/src/main/kotlin/org/jetbrains/jewel/samples/standalone/view/component/Banners.kt:7]: image

😄

**Review by rock3r — APPROVED** [review]

**Comment by rock3r** [comment]
Actually, the banner border are missing after #699, so undoing the approval for now

**Review by rock3r — APPROVED** [review]
Ah, no PR I guess :P

**Comment by hamen** [comment]
NobodyGotTimeForThatAIntnobodyGotTimeForThatGIF

**Pull request merged**